src/ResourceManager/DeploymentManager/Commands.DeploymentManager/Az.DeploymentManager.psd1

```diff
@@ -91,7 +91,11 @@ CmdletsToExport = 'Get-AzDeploymentManagerArtifactSource',
                'Get-AzDeploymentManagerServiceUnit', 
                'New-AzDeploymentManagerServiceUnit', 
                'Set-AzDeploymentManagerServiceUnit', 
-               'Remove-AzDeploymentManagerServiceUnit'
+               'Remove-AzDeploymentManagerServiceUnit',
```

Comment:
youlll need o create alias mappings for these using : https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/create-alias-mappings.md